### PR TITLE
First Milestone: Hello Microverse Pull Request 

### PR DESCRIPTION
**Pull request by HussenYesufAlli**
In this project I:
add index.html file that includes a header
add style.css file that includes one class with styles for my header
update readme file according to the template
add license file that use the MIT license

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,122 @@
...
+### Key Features <a name="key-features"></a>
+
+- **[Github workFlows]**
+- **[Linters Configuration Setup]**
+- **[Follow HTML and CSS best practice]**
+- **[Display Header Text]**
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## :rocket: Live Demo <a name="live-demo"></a>
```
Comment (tufoinnkuo10) [inline README.md:58]: - [ ] Great job for adding a logo, deploying your project and using the correct readme template. please always ensure that all the main headers from this [README template](https://github.com/microverseinc/readme-template) are available. the only optional one is the faq. you are missing the
[💻 Getting Started](https://github.com/microverseinc/readme-template#getting-started)
 [🤝 Contributing](https://github.com/microverseinc/readme-template#contributing),
[⭐️ Show your support](https://github.com/microverseinc/readme-template#support),
[🙏 Acknowledgements](https://github.com/microverseinc/readme-template#acknowledgements)
having them with little necessary information under them will make your README more professional. nice work.

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @HussenYesufAlli 

great job so far. There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ gitignore file well used


Required Changes ♻️
Check the comments under the review

tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

File: README.md (hunk 1 above)
Comment (Abenezer-Tilahun) [reply README.md:58]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @HussenYesufAlli  
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git Hub-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**